persist/persist_test.go

```diff
@@ -0,0 +1,25 @@
+package persist
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/build"
+)
+
+// TestIntegrationRandomSuffix checks that the random suffix creator creates
+// valid files.
+func TestIntegrationRandomSuffix(t *testing.T) {
```

Comment:
Hmm. I feel like the useful distinction is not between unit vs. integration, but rather exported-only vs. non-exported.